content/applications/finance/accounting/payments/follow_up.rst

```diff
@@ -14,44 +14,61 @@ Configuration
 
-It is also possible to automatically send a reminder by enabling the :guilabel:`Auto Execute`
-option, within a specific :guilabel:`Follow-Up Action`.
+You can use a pre-filled template for your messages by selecting a :guilabel:`Content Template`. To
+change the template used, hover over the the field and click the :guilabel:`-->`. If enabled, SMS
+messages have a specific :guilabel:`Sms Template` field.
+
+It is possible to automatically send a reminder by enabling the :guilabel:`Automatic` option, and
+attaching the *open* invoice(s) by enabling :guilabel:`Attach Invoices`, within a specific follow-up
+action.
+
+By clicking on the :guilabel:`Activity` tab, it's possible to schedule activities (tasks). That way,
+when the follow-up is triggered, an activity is automatically scheduled. To do so, enable
+:guilabel:`Schedule Activity`, and select a :guilabel:`Responsible` person for the task. Choose an
+:guilabel:`Activity Type`, and enter a :guilabel:`Summary` on how to handle the activity, if
+desired.
 
 .. tip::
    To send a reminder before the actual due date is reached, set a negative number of due days.
 
 Follow-up reports
 =================
 
-The overdue invoices you need to follow up on are available in :menuselection:`Accounting -->
-Customers --> Follow-up Reports`. From there, you  can identify all the customers that have late
-unpaid invoices.
+Overdue invoices you need to follow up on are available in :menuselection:`Accounting --> Customers
+--> Follow-up Reports`. By default, Odoo filters by :guilabel:`Customer Invoices` that are
+:guilabel:`In need of action`.
+
+When selecting an invoice, you can see all of the customer's unpaid invoices (overdue or not), with
+due dates of late invoices appearing in red. You can exclude invoices from a reminder by clicking
+:guilabel:`Exclude from Follow-ups`. You can set either :guilabel:`Automatic` or :`Manual` reminders
+as well as a :guilabel:`Responsible` person for that customer.
 
-When you open a customer, you see all their unpaid invoices (overdue or not). The due dates of the
-late invoices appear in red. Select the invoices that are not late yet in the :guilabel:`Excluded`
-column to exclude them from the reminder you send.
+To send reminders, click on :guilabel:`Follow up`, and select the action(s) you want to perform
+from:
 
-It is up to you to decide how to remind your customer. You can select :guilabel:`Print Letter`,
-:guilabel:`Send By Email`, :guilabel:`Send By Post`, :guilabel:`Send By SMS`. Then, click on
-:guilabel:`Done` to view the next follow-up that needs your attention.
+- :guilabel:`Print`;
+- :guilabel:`Email`;
+- :guilabel:`Sms`;
+- :guilabel:`By post`.
+
+You can :guilabel:`Attach Invoices` and change the templates of the content from this view. When
+done, click :guilabel:`Send` or :guilabel:`Send & Print`.
 
 .. note::
    - The contact information available on the invoice or on the contact form is used to send the
```

Comment:
- The contact information on the invoice or the contact form is used to send the reminder.